### DISCUSSION TITLE
Carousel prop `align` to handle a mixture between `start` and `end`

### DISCUSSION THREAD

**Discussion by ylkhayat** [mantinedev/mantine#3574] [Feature requests]
Is it possible to implement something that doesn't scroll more than it should, like mainly allow scrolling from the first element being available at the very beginning and the last element being available at the very end.

If we have 5 elements
[(1)  (2)  (3) (] `align="start"`
[)  (2) (3) (4)]
[(2) (3) (4) (]
[) (3) (4) (5)] `align="end"`

**Comment by rtivital** [comment]
I believe it should be possible to configure embla this way. Although I do not really know which props combination you should use, try searching through embla docs – https://www.embla-carousel.com/

**Reply by ylkhayat** [reply]
Thanks! I will give it a try and post here if i found a decent way to handle it 🎉

**Comment by StanislavRassolenko** [comment]
In case anyone else is looking for the solution, now it's possible to use `containScroll = "trimSnaps"` that does exactly that.
```
<Carousel containScroll={'trimSnaps'}>
...
</Carousel>
```
